modules/stats.php

```diff
@@ -545,21 +724,27 @@ function stats_configuration_load() {
 			if ( isset( $_POST["role_$role"] ) && $_POST["role_$role"] )
 				$options['roles'][] = $role;
 
-		$options['count_roles'] = array();
+			$options['count_roles'] = array();
 		foreach ( get_editable_roles() as $role => $details )
 			if ( isset( $_POST["count_role_$role"] ) && $_POST["count_role_$role"] )
 				$options['count_roles'][] = $role;
 
-		stats_set_options( $options );
+			stats_set_options( $options );
```

Comment:
This line doesn't belong to the previous `if`, nor to the `foreach`,  it shouldn't be indented.